### PR TITLE
bump miniquad version to 0.3.0-alpha.29

### PR DESCRIPTION
**Pull request by extrawurst**

### CODE REVIEW

**Comment by emilk** [comment]
Did you check if `doc/gl.js` also needs updating? (We should have some procedure for this, but we don't)

**Comment by extrawurst** [comment]
ha good point, it does

**Comment by extrawurst** [comment]
I was not aware that we copy over the `gl.js` from miniquad.

**Comment by extrawurst** [comment]
`gl.js` updated

**Comment by not-fl3** [comment]
With not pinned miniquad version it is slightly easier to get a mismatched gl.js/miniquad version, I would consider keeping it pinned just in case

File: Cargo.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ include = [
 
 [dependencies]
 egui = "0.11.0"
-miniquad = { version = "=0.3.0-alpha.28" }
+miniquad = { version = "0.3.0-alpha.29" }
```
Comment (emilk) [inline Cargo.toml:22]:
```suggestion
miniquad = { version = "=0.3.0-alpha.29" }
```

we should pin while using pre-releases as they can contain breaking changes

**Review by emilk — CHANGES_REQUESTED** [review]

**Comment by extrawurst** [comment]
@not-fl3 speaking of **pre-releases** - is there any plan to release 0.3.0 ? :P

**Comment by emilk** [comment]
Thanks!

**Comment by not-fl3** [comment]
> @not-fl3 speaking of **pre-releases** - is there any plan to release 0.3.0 ? :P

oh, that's a good question. 
Realistically speaking for the next few weeks I'll be too busy working on macroquad stable release and probably will just use alpha miniquad there :(

**Pull request merged**